airflow/data/agencies.yml

```diff
@@ -1409,7 +1433,7 @@ westcat:
 yolobus:
   agency_name: Yolobus
   feeds:
-    - gtfs_schedule_url: http://iportal.sacrt.com/GTFS/Yolobus/google_transit.zip
+    - gtfs_schedule_url: http://www.yolobus.com/GTFS/google_transit.zip
```

Comment:
this looks like the same data as is in the iportal.sacrt.  Is SAC-RT mirroring yolobus.com? Or visa versa? (aka who is the publisher?)

Reply:
Yolobus is the publisher and SacRT is mirroring. Yolobus made it clear that they consider their link to be the ultimate source of truth.